packages/snaps-controllers/package.json

```diff
@@ -16,8 +16,9 @@
   ],
   "scripts": {
     "test:prepare": "yarn mkdirp test/fixtures && ./scripts/generate-fixtures.sh",
-    "test": "yarn test:prepare && jest && yarn posttest",
+    "test": "yarn test:prepare && jest && yarn posttest && yarn test:browser",
```

Comment:
Note: `test:browser` must run after `posttest`, because `wdio` deletes the coverage folder for some reason, breaking `jest-it-up`.

Reply:
Can we collect coverage from `wdio` or no?

Reply:
Yes. I will do this as part of #1234.